packages/schematics/angular/workspace/schema.json

```diff
@@ -33,9 +33,9 @@
       "x-user-analytics": 14
     },
     "strict": {
-      "description": "Creates a workspace with stricter TypeScript compiler options.",
+      "description": "Creates a workspace with stricter TypeScript and Angular compiler options.",
       "type": "boolean",
-      "default": false
+      "default": true
```

Comment:
Should we consider deprecating this option for 10?

Reply:
I think that might be a good idea.

@IgorMinar, thoughts?

Reply:
let's write up a doc proposing that we enable the `strict` mode in CLI projects

The doc should contain at least these:

- the current default
- what is strict mode
- the impact of this change on
  - new projects
  - existing projects (what's the migration path?)
  - existing ecosystem
    - libraries
    - tutorial / blog posts/ documentation
    - workshop materials
- the proposed rollout plan

once we have all of this, we can review it, and then share it with GDEs, and the community for feedback to validate that our plan is solid before we actually change the default in the CLI.